bugsnag-plugin-android-ndk/src/main/jni/event.h

```diff
@@ -192,6 +205,9 @@ typedef struct {
   int crumb_first_index;
   bugsnag_breadcrumb breadcrumbs[BUGSNAG_CRUMBS_MAX];
 
+  int thread_count;
+  bsg_thread threads[BUGSNAG_THREADS_MAX];
```

Comment:
These fields can't be added without a migration, particularly since its in the middle of the structure instead of the end.

Reply:
Yup, migration is in a separate task. Will move the new data to the end of the struct to make that easier.